.gitignore

```diff
@@ -23,4 +23,4 @@ node_modules
 # Generated js bundles
 /public/webpack/*
 
```

Comment:
We should add `generatedComponentConfig.js` to this file

Reply:
If we add the generateComponentConfig, it cannot be referenced within the site config, which then loads it into site.

Reply:
![whompwhomp](https://media1.tenor.com/images/4180e99f9cf19ed9f7f80a2e29b4f4b8/tenor.gif?itemid=4920633)